docs/operator-manual/upgrading/3.2-3.3.md

```diff
@@ -2,6 +2,31 @@
 
 ## Breaking Changes
 
+### ApplicationSet CRD exceeds the size limit for client-side apply
+
+In this version, the ApplicationSet CRD exceeds the size limit for client-side apply, meaning that the previous upgrade methods will result in `The CustomResourceDefinition "applicationsets.argoproj.io" is invalid: metadata.annotations: Too long: may not be more than 262144 bytes`
+
+In order to avoid this issue, it is now required to upgrade Argo CD using Server Side Apply (SSA) along with `resolve conflicts`.
+SSA does not store the ApplicationSet fields in `last-applied` annotation, thus not impacted by the annotation size limitation.
+The `--force-conflicts` flag allows the apply operation to take ownership of fields that may have been previously managed by other tools (such as Helm or a previous `kubectl apply`). This is necessary for upgrades. Note that any custom modifications you've made to fields that are defined in the Argo CD manifests (like `affinity`, `env`, or `probes`) will be overwritten. However, fields not specified in the manifests (like `resources` limits/requests or `tolerations`) will be preserved.
+
+#### Upgrading Argo CD which is managing itself 
+
+When Argo CD is upgraded using an Argo CD Application, it is required to add 
+``` yaml
+    syncOptions:
+    - ServerSideApply=true
+```
+to that Application. With this configuration, Argo CD will add the `resolve conflicts` option automatically.   
```

Comment:
```suggestion
With this configuration, Argo CD will add the `resolve conflicts` option automatically.   
```

Reply:
`to that Application` is the continuation of the sentence.